### PR TITLE
Feature/tri 608

### PR DESCRIPTION
**Pull request by ds-ext-kmassalski**

### CODE REVIEW

File: irs-api/src/main/java/net/catenax/irs/semanticshub/SemanticsHubFacade.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+     */
+    @Cacheable(value = SEMANTICS_HUB_CACHE_NAME, key = "#urn")
+    public String getModelJsonSchema(final String urn) {
+        return this.semanticsHubClient.getModelJsonSchema(urn);
+    }
+
+    /**
+     * Clearing cache from all values
+     */
+    @SuppressWarnings("PMD.UncommentedEmptyMethodBody")
```
Comment (ds-jkreutzfeld) [inline irs-api/src/main/java/net/catenax/irs/semanticshub/SemanticsHubFacade.java:43]: You could put a log.debug in the body :D

**Review by ds-jkreutzfeld — CHANGES_REQUESTED** [review]
Also, please change the leaked secret ;)

File: irs-api/src/main/java/net/catenax/irs/semanticshub/SemanticsHubFacade.java (hunk 1 above)
Comment (ds-ext-kmassalski) [reply irs-api/src/main/java/net/catenax/irs/semanticshub/SemanticsHubFacade.java]: Done

File: irs-api/src/main/java/net/catenax/irs/semanticshub/SchemaModel.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+/**
+ * Common Schema released models
+ */
+@AllArgsConstructor
+@Getter
+@SuppressWarnings("PMD.CommentSize")
+enum SchemaModel {
+//    TODO(kmassalski): currently returning 500 status
+//    ASSEMBLY_PART_RELATIONSHIP("urn:bamm:com.catenax.assembly_part_relationship:1.0.0#AssemblyPartRelationship"),
+    SERIAL_PART_TYPIZATION("urn:bamm:io.catenax.serial_part_typization:1.0.0#SerialPartTypization");
```
[the remark this answers is not in the document]
Comment (ds-ext-kmassalski) [reply irs-api/src/main/java/net/catenax/irs/semanticshub/SchemaModel.java]: Yep agree that storing this in code was not a good idea - I moved it to application.yaml, as its not possible to retrieve URN's from DTS/Semantics.

**Comment by ds-ext-kmassalski** [comment]
`Also, please change the leaked secret ;)`
- yesterday I removed this commit from git history - is there anything more to do?

**Review by ds-jkreutzfeld — APPROVED** [review]

**Pull request merged**